### ISSUE TITLE
Create a new React component for rendering information regarding the various referenda tracks

### ISSUE THREAD

**Issue by alfarok** [w3f/polkadot-wiki#4346]
It would be very informative to display the various referenda tracks and some information about each one in the wiki docs.

The `consts.referenda.tracks` RPC endpoint returns the following information for all tracks:

```js
[
  [
    0
    {
      name: root
      maxDeciding: 1
      decisionDeposit: 3,333,333,333,300,000
      preparePeriod: 1,200
      decisionPeriod: 201,600
      confirmPeriod: 14,400
      minEnactmentPeriod: 14,400
      minApproval: {
        Reciprocal: {
          factor: 222,222,224
          xOffset: 333,333,335
          yOffset: 333,333,332
        }
      }
      minSupport: {
        LinearDecreasing: {
          length: 100.00%
          floor: 0.00%
          ceil: 50.00%
        }
      }
    }
  ],
  ...
]
```

**Comment by alfarok** [comment]
First pass:

![image](https://user-images.githubusercontent.com/13341935/215531744-b1eb9f22-305b-42b2-bfbb-d30b1988588f.png)

**Comment by alfarok** [comment]
Addressed in #4378
